Sprint23 eit

amal-ghamdi opened the pull request:
closes #16

amal-ghamdi commented:
@jakobsj, can you please take a look at the EIT code. 

@babakmaboudi, can you please take a look at the updated code to make sure my updates are alright. 

I added few comments about it (in the PR description). Many thanks for both of you!

jakobsj commented on EIT/EIT.py:
```diff
@@ -137,64 +159,71 @@ def Heaviside(func):
-# We copy the PDE1 for the rest of the PDE problems 
+# for the first PDE problems we specify to reuse the factorization of the lhs 
+# for the rest of the PDE models
+PDE1 = SteadyStateLinearFEniCSPDE(
```
I suggest splitting to have a single input on each line to make all inputs appear equally important/visible. This makes it also visually simpler in my opinion.

amal-ghamdi replied:
done ✅

jakobsj reviewed (approved):
Nicely done, the code has been simplified and easier to read, based on the new changes to CUQIpy as well as the way you have restructured, for example doing all imports at the beginning.  Only made a small suggestion to further split multi-line lists of input arguments onto separate lines.  Approving.

amal-ghamdi commented:
Many thanks @jakobsj for your review and for your comment! I update the PDE initialization line to have input in separate lines.

amal-ghamdi commented:
@jakobsj and @babakmaboudi I appreciate it if you can take a look at the updateed PR. Babak added the plotting files and I restructured them in 2 scripts `plot_paper_figures.py` and `figures_util.py`. Running `plot_paper_figures.py` after generating the sample files will produce paper figures 6, 7, 8 and 9 in the paper. I updated the overleaf as well with the latest EIT figure we have using the fixed seed in `EIT.py`. Readme instructions were also updated. Many thanks!

jakobsj reviewed (approved):
Nicely organised and clear instructions, well done!
